src/set_theory/surreal/basic.lean

```diff
@@ -75,59 +76,52 @@ theorem numeric_rec {C : pgame → Prop}
+  { exact not_lf.2 (le_trans h₁ h₂) (lf_of_lt (hx _ _)) },
+  { exact IHxr _ _ (oyr _) (lf_move_right_of_le _ h₁) (lf_move_right_of_le _ h₂) },
 end
 
-theorem le_of_lt {x y : pgame} (ox : numeric x) (oy : numeric y) (h : x < y) : x ≤ y :=
-not_lt.1 (lt_asymm ox oy h)
+theorem le_of_lf {x y : pgame} (ox : numeric x) (oy : numeric y) (h : x ⧏ y) : x ≤ y :=
+not_lf.1 (lf_asymm ox oy h)
 
-/-- `<` is transitive when both sides of the left inequality are numeric -/
-theorem lt_trans {x y z : pgame} (ox : numeric x) (oy : numeric y) (h₁ : x < y)
-  (h₂ : y < z) : x < z :=
-lt_of_le_of_lt (le_of_lt ox oy h₁) h₂
+theorem lt_of_lf {x y : pgame} (ox : numeric x) (oy : numeric y) (h : x ⧏ y) : x < y :=
+(lt_or_fuzzy_of_lf h).resolve_right (not_fuzzy_of_le (le_of_lf ox oy h))
 
-/-- `<` is transitive when both sides of the right inequality are numeric -/
-theorem lt_trans' {x y z : pgame} (oy : numeric y) (oz : numeric z) (h₁ : x < y)
-  (h₂ : y < z) : x < z :=
-lt_of_lt_of_le h₁ (le_of_lt oy oz h₂)
+theorem lf_iff_lt {x y : pgame} (ox : numeric x) (oy : numeric y) : x ⧏ y ↔ x < y :=
+⟨lt_of_lf ox oy, lf_of_lt⟩
 
-/-- On numeric pre-games, `<` and `≤` satisfy the axioms of a partial order (even though they
-don't on all pre-games). -/
-theorem lt_iff_le_not_le {x y : pgame} (ox : numeric x) (oy : numeric y) :
-  x < y ↔ x ≤ y ∧ ¬ y ≤ x :=
-⟨λ h, ⟨le_of_lt ox oy h, not_le.2 h⟩, λ h, not_le.1 h.2⟩
+theorem not_fuzzy {x y : pgame} (ox : numeric x) (oy : numeric y) : ¬ fuzzy x y :=
+λ h, not_lf.2 (le_of_lf ox oy (lf_of_fuzzy h)) h.2
 
 theorem numeric_zero : numeric 0 :=
 ⟨by rintros ⟨⟩ ⟨⟩, ⟨by rintros ⟨⟩, by rintros ⟨⟩⟩⟩
 theorem numeric_one : numeric 1 :=
 ⟨by rintros ⟨⟩ ⟨⟩, ⟨λ x, numeric_zero, by rintros ⟨⟩⟩⟩
 
 theorem numeric.neg : Π {x : pgame} (o : numeric x), numeric (-x)
-| ⟨l, r, L, R⟩ o := ⟨λ j i, lt_iff_neg_gt.1 (o.1 i j), λ j, (o.2.2 j).neg, λ i, (o.2.1 i).neg⟩
+| ⟨l, r, L, R⟩ o := ⟨λ j i, neg_lt_iff.2 (o.1 i j), λ j, (o.2.2 j).neg, λ i, (o.2.1 i).neg⟩
 
-/-- For the `<` version, see `pgame.move_left_lt`. -/
-theorem numeric.move_left_le {x : pgame} (o : numeric x) (i : x.left_moves) :
-  x.move_left i ≤ x :=
-le_of_lt (o.move_left i) o (pgame.move_left_lt i)
+theorem numeric.move_left_lt {x : pgame} (o : numeric x) (i) : x.move_left i < x :=
+lt_of_lf (o.move_left i) o (pgame.move_left_lf i)
+theorem numeric.move_left_le {x : pgame} (o : numeric x) (i) : x.move_left i ≤ x :=
+(o.move_left_lt i).le
 
-/-- For the `<` version, see `pgame.lt_move_right`. -/
-theorem numeric.le_move_right {x : pgame} (o : numeric x) (j : x.right_moves) :
-  x ≤ x.move_right j :=
-le_of_lt o (o.move_right j) (pgame.lt_move_right j)
+theorem numeric.lt_move_right {x : pgame} (o : numeric x) (j) : x < x.move_right j :=
+lt_of_lf o (o.move_right j) (pgame.lf_move_right j)
+theorem numeric.le_move_right {x : pgame} (o : numeric x) (j) : x ≤ x.move_right j :=
+(o.lt_move_right j).le
 
-theorem add_lt_add
+theorem add_lf_add
```

Comment:
This lemma also looks a bit weird right now. In fact, it can be generalized to
```lean
theorem add_lf_add_of_lf_of_lt {w x y z : pgame.{u}} (hwx : w ⧏ x) (hyz : y < z) : w + y ⧏ x + z := sorry
```
which doesn't depend on any `numeric` hypotheses. Again, since this PR is already huge as is, I'll leave it like this and do this change on a subsequent PR.

Reply:
If you like you can add a `TODO` to this effect, so it can't be forgotten. But it's not necessary.